coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +168,29 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        def get_from_docstyle_settings(*args):
+            values = list()
+            for req_setting in args:
+                setting = dict(
+                    docstyle_settings.contents.items()).get(req_setting)
+                try:
+                    setting = setting.value
+                except AttributeError:
+                    setting = ""
+                values.append(setting)
+            return values
```

Comment:
hm this looks like some `get_values_or_default`, maybe you should just hardcode that below?
```
param_start_value = section.get("param_start", "")
```